litrpc/lit-accounts.proto

```diff
@@ -25,6 +25,18 @@ service Accounts {
     */
     rpc UpdateAccount (UpdateAccountRequest) returns (Account);
 
+    /* litcli: `accounts creditaccount`
+    CreditAccount increases the balance of an existing account in the account
+    database.
+    */
+    rpc CreditAccount (UpdateAccountBalanceRequest) returns (Account);
+
+    /* litcli: `accounts debitaccount`
+    DebitAccount decreases the balance of an existing account in the account
+    database.
+    */
+    rpc DebitAccount (UpdateAccountBalanceRequest) returns (Account);
```

Comment:
I know the `UpdateAccount` method returns an `Account` only, but it is generally better practice to instead define new messages here: 
`CreditAccountRequest`, `CreditAccountResponse` etc etc

so that if how we handle the requests/responses for either of these ever changes/get added to, then it is easy to extend the individual messages rather than breaking the API

Reply:
Great point! Updated this to address your feedback.